docs/roman/data_products/science_products.rst

```diff
@@ -139,3 +139,39 @@ The calibrated products are the result of an average over all integrations (``ca
  - border_ref_pix_right: Copy of original border reference pixels, on right (from viewers perspective).
  - border_ref_pix_top: Copy of original border reference pixels, on the top (from viewers perspective).
  - border_ref_pix_bottom: Copy of original border reference pixels, on the bottom (from viewers perspective).
+
+
+Resampled 2-D data: ``i2d``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Images and spectra that have been resampled by the :ref:`resample <resample_step>` step use a
+different set of data arrays than other science products. Resampled 2-D images are stored in
+``i2d`` products.
+The ASDF structure for ``i2d`` products is as follows:
+
++----------------------+----------+------------+---------------------------+-------------------------------+
+| data array           |          | Data Type  | Units                     | Dimensions                    |
++======================+==========+============+===========================+===============================+
+|  data                | Required | float32    | e\ :sup:`-`/ s            |  nrows x ncols                |
++----------------------+----------+------------+---------------------------+-------------------------------+
+|  context             | Required | uint32     | N/A                       |  2 x nrows x ncols            |
++----------------------+----------+------------+---------------------------+-------------------------------+
+|  err                 | Required | float32    | e\ :sup:`-`/ s            |  nrows x ncols                |
++----------------------+----------+------------+---------------------------+-------------------------------+
+|  weight              | Required | float32    | N/A                       |  nrows x ncols                |
++----------------------+----------+------------+---------------------------+-------------------------------+
+|  var_poisson         | Required | float32    | (e\ :sup:`-`/ s)\ :sup:`2`|  nrows x ncols                |
++----------------------+----------+------------+---------------------------+-------------------------------+
+|  var_rnoise          | Required | float32    | (e\ :sup:`-`/ s)\ :sup:`2`|  nrows x ncols                |
++----------------------+----------+------------+---------------------------+-------------------------------+
+|  var_flat            | Required | float32    | (e\ :sup:`-`/ s)\ :sup:`2`|  nrows x ncols                |
++----------------------+----------+------------+---------------------------+-------------------------------+
```

Comment:
Did you check the units here?  The flux calibration step is supposed to make sure the units are MJy/sr (or the square of that for the variances), but we may have errors, of course.  I had understood the context image to be N x nrows x ncols, where N depends on the number of input images, and is 1 for sky cells with fewer than 32 input images.  Not true?

Reply:
I must have looked at an old file. I've updated the units. 

for the number of planes in the context image I thought as you said, it should be the number of input images modulo 32. I in the output I see
context (NDArrayType): shape=(2, 9191, 8642), dtype=uint32 # Context Data
The number of input images was 8 (not all overlapping). 
Maybe a bug?